src/test/scala/mesosphere/marathon/core/task/bus/TaskStatusUpdateTestHelper.scala

```diff
@@ -22,17 +22,19 @@ class TaskStatusUpdateTestHelper(val operation: InstanceUpdateOperation, val eff
     case InstanceUpdateOperation.MesosUpdate(_, marathonTaskStatus, mesosStatus, _) => mesosStatus
     case _ => throw new scala.RuntimeException("the wrapped stateOp os no MesosUpdate!")
   }
+
+  private[this] def instanceFromOperation: Instance = operation match {
```

Comment:
I think you can remove it. It has only one use case in `def updatedInstance` and since we checkout only for `MesosUpdate` we can to that in `updatedInstance`.